src/app/inventory/StoreBucket.tsx

```diff
@@ -79,9 +79,14 @@ function mapStateToProps() {
           items = [...items, ...findItemsByBucket(otherStore, bucket.hash)];
         }
       }
-      const currentStore = currentStoreSelector(state)!;
+      const currentStore = currentStoreSelector(state);
+      // TODO: When we switch accounts this suffers from the "zombie child" problem where the redux store has already
+      // updated (so currentStore is cleared) but the store from props is still around because its redux subscription
+      // hasn't fired yet.
```

Comment:
This is a separate but somewhat related problem related to how Redux works in general. I'm not super sure how to fix it. For more reading: https://react-redux.js.org/api/hooks#stale-props-and-zombie-children